functions/__sf_section_docker.fish

```diff
@@ -0,0 +1,51 @@
+#
+# Docker
+#
+# Current Docker version and Machine name.
+
+function __sf_section_docker -d "Display docker version and machine name"
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_DOCKER_SHOW true
+    __sf_util_set_default SPACEFISH_DOCKER_PREFIX "is "
+    __sf_util_set_default SPACEFISH_DOCKER_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+    __sf_util_set_default SPACEFISH_DOCKER_SYMBOL "🐳 "
+    __sf_util_set_default SPACEFISH_DOCKER_COLOR cyan
+    __sf_util_set_default SPACEFISH_DOCKER_VERBOSE_VERSION false
+
+    # ------------------------------------------------------------------------------
+    # Section
+    # ------------------------------------------------------------------------------
+
+    [ $SPACEFISH_DOCKER_SHOW = false ]; and return
+
+    # Show Docker version only if docker is installed
+    type -q docker; or return
+
+    # Show docker version only when pwd has dockerfile or docker-compose.yml or COMPOSE_FILE
+    [ -f ./Dockerfile -o -f ./docker-compose.yml ]; or test "$COMPOSE_FILE"; or return
```

Comment:
nitpick: Just for easier readability and to better match [the other times](https://github.com/matchai/spacefish/blob/3479f08882cd7473ead1e90f10cc35a86d7874b4/functions/__sf_section_golang.fish#L29-L36) where we check multiple filenames for early returns, I suggest breaking this into a multi-line test. We tend to use short-hand notation for test when there is only one argument. 🔨 

```suggestion
	if not test -f Dockerfile \
		-o -f docker-compose.yml \
		-o -f $COMPOSE_FILE
		return
	end
```